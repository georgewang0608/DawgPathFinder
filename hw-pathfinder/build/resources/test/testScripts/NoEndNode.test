# Find a path with Dijkstra's algorithm when
# there does not exist an end node for the path to be found in the graph

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B

AddEdge simpleGraph A B 10.0

FindPath simpleGraph A D
