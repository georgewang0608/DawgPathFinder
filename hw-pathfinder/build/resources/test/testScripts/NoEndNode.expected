# Find a path with Dijkstra's algorithm when
# there does not exist an end node for the path to be found in the graph

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph

added edge 10.000 from A to B in simpleGraph

unknown: D
